src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -34,6 +32,30 @@
                   :checked?            (some #(= % chain-id) @network-preferences)
                   :on-change           on-change}})
 
+(defn- find-insertion-index
+  [data chain-id loading-suggested-routes?]
+  (let [network           (utils/id->network chain-id)
+        new-item-priority (network-priority network)]
+    (or (->> data
```

Comment:
what is `data` here btw?

Reply:
Sorry, some leftover, I updated with more descriptive naming for functions and variables!